src/Services/EncounterService.php

```diff
@@ -16,8 +16,13 @@
 
 namespace OpenEMR\Services;
 
+
 use Particle\Validator\Validator;
 
+require_once($GLOBALS['fileroot'] . "/library/forms.inc");
+require_once($GLOBALS['fileroot'] . "/library/encounter.inc");
```

Comment:
although not pretty, recommend direct paths in this setting:
```
require_once(dirname(__FILE__) . "/../../library/forms.inc");
require_once(dirname(__FILE__) . "/../../library/encounter.inc");
```

Reply:
Hello @bradymiller, I am fixing this right away. In between why do you prefer the later?

Reply:
general rule of thumb to prevent weird bugs down road:
In libraries/classes use direct paths like above
In scripts where including the globals.php, then use the $GLOBALS stuff for paths

Reply:
Thanks for this